### PR TITLE
sanitisation and testing of DEL+UPD+FUNC_REQ

### PR DESCRIPTION
**Pull request by alapan-sau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+# Social Media DB
+
+A simple database application designed with miniworld as a social media, with CLI interface, which allows users to perform all the tasks in an actual social media such as sharing posts, reacting and commenting to posts, update profile details, etc.
+
+## Steps to Run
+
+
+```shell 
+ mysql -h <local host> -u <username> --port=<port> -p > SocialMediaDB.sql
+ python3 app.py
```
Comment (NirmalManoj) [inline README.md]: Shouldn't it be `-p < SocialMediaDB.sql` or will this also work?

File: app.py [hunk 2]
```diff
@@ -103,10 +103,13 @@ def search():
...
+    else:
+        print("Invalid Domain Error")
+        return
 
     try:
         query = "SELECT * FROM %s WHERE %s REGEXP '%s'" % (
-            search_type, search_field, search_key)
+            search_type, search_field, search_key, )
         r = cur.execute(query)
         if r == 0:
```
Comment (NirmalManoj) [inline app.py:114]: You are not using parameterized queries?

Comment (NirmalManoj) [reply app.py:114]: Nevermind, just realized that this is not a user input.

**Review by NirmalManoj — CHANGES_REQUESTED** [review]
I haven't tested it, will test it later. Apart from the one comment I made, this looks good.

File: README.md (hunk 1 above)
Comment (NirmalManoj) [reply README.md]: @pavani-17

**Review by NirmalManoj — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Radha999) [reply README.md]: Oops... No it wont work... It sud be <

**Pull request merged**